frontend/src/component/executiveDashboard/Gauge/Gauge.tsx

```diff
@@ -0,0 +1,195 @@
+import { VFC } from 'react';
+import { Box, useTheme } from '@mui/material';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+
+const polarToCartesian = (
+    centerX: number,
+    centerY: number,
+    radius: number,
+    angleInRadians: number,
+) => {
+    return {
+        x: centerX + radius * Math.cos(angleInRadians),
+        y: centerY + radius * Math.sin(angleInRadians),
+    };
+};
+
+const degreesToRadians = (degrees: number) => degrees * (Math.PI / 180);
+
+const normalizeValue = (value: number, min: number, max: number) =>
+    (value - min) / (max - min);
+
+const describeArc = (radius: number, startAngle: number, endAngle: number) => {
+    const start = polarToCartesian(0, 0, radius, endAngle);
+    const end = polarToCartesian(0, 0, radius, startAngle);
+    const largeArcFlag = endAngle - startAngle <= Math.PI ? '0' : '1';
+    const d = [
+        'M',
+        start.x,
+        start.y,
+        'A',
+        radius,
+        radius,
+        0,
+        largeArcFlag,
+        0,
+        end.x,
+        end.y,
+    ].join(' ');
+    return d;
+};
+
+const GaugeLines = () => {
+    const theme = useTheme();
+    const startRadius = 0.875;
+    const endRadius = 1.14;
+    const lineWidth = 0.1;
+    const lineBorder = 0.05;
+
+    const angles = [180 + 50, 180 + 90 + 40];
+
+    return (
+        <>
+            {angles.map(degreesToRadians).map((angle) => {
+                const start = polarToCartesian(0, 0, startRadius, angle);
+                const end = polarToCartesian(0, 0, endRadius, angle);
+
+                return (
+                    <>
```

Comment:
Don't we need to encapsulate these with svg?

EDIT: I see they are encapsulated further down. But maybe a comment that this needs to be used in the context of an svg tag.